.github/workflows/build-publish-image.yml

```diff
@@ -9,7 +9,7 @@ on:
       - '**'
 
 env:
-  DOCKER_IMAGE: ghcr.io/iot-inspector/unblob:latest
+  DOCKER_IMAGE: ghcr.io/onekey-sec/unblob:latest
```

Comment:
Is it ready to be merged ? Does this registry exist ?

Reply:
Yes, it is the registry we inherited through renaming the organization https://github.com/onekey-sec/unblob/pkgs/container/unblob